crates/bevy_winit/src/accessibility.rs

```diff
@@ -99,7 +81,7 @@ fn update_accessibility_nodes(
     )>,
     node_entities: Query<Entity, With<AccessibilityNode>>,
 ) {
-    if !accessibility_requested.load(Ordering::SeqCst) {
+    if !accessibility_requested.get() || !manage_accessibility_updates.get() {
```

Comment:
This will be slightly more efficient (and clearer) if we split this out into a run condition.

Reply:
Thanks, good suggestion. Fixed.